### PR TITLE
Cms/352 Create “edit project” page

### PR DESCRIPTION
**Pull request by foxyfiction**
Add edit-project.html
Add edit-project-style.css

### CODE REVIEW

File: frontend/css/edit-project-style.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ul {
+    font-size: 14px;
+    font-weight: 400;
+}
+.menu__item {
+    display: table-cell;
+    padding-top: 22px;
+    float: left;
+    list-style: none;
+    margin: 0px 2px;
```
Comment (IgorZiegler) [inline frontend/css/edit-project-style.css]: You should omit `px` when value is 0

File: frontend/css/edit-project-style.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+.menu {
+    display: table;
+    position: fixed;
+    top: 0;
+    background-color: white;
+    width: 100vw;
+    height: 60px;
+    box-shadow: 0 0 10px rgba(0,0,0,0.1);
+    font-family: Geneva, Arial, Helvetica, sans-serif;
```
Comment (IgorZiegler) [inline frontend/css/edit-project-style.css]: Move it to the body

File: frontend/html/edit-project.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <div class="menu">
+        <div class="left-side">
+            <a href="">Dashboard</a>
+            <span> &nbsp; ⟶ &nbsp; </span>
+            <a href="">Мой проект</a>
+        </div>
+        <div class="right-side">
+            <ul>
+                <li class="menu__item"><a href="">Preview</a></li>
+                <li class="menu__item menu__item_font-weight_bold"><a href="">Publish</a></li>
```
Comment (maxmalov) [inline frontend/html/edit-project.html]: No need to specify whole css in the modifier name, it should be short and informative, for example bold/active/highlighted.

Also do not mix `-` and `_` in ccs names. Usually in BEM `-` used for word separation, `__` for item separation and `--` for modifier separation

**Comment by maxmalov** [comment]
@foxyfiction Is this PR actual or can be closed?

**Pull request closed without merging**